metadata.json

```diff
@@ -77,6 +60,14 @@
     {
       "name": "puppetlabs-concat",
       "version_requirement": ">= 2.0.0 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs-apt",
+      "version_requirement": ">= 5.0.0 < 7.0.0"
+    },
+    {
+      "name": "puppetlabs-yumrepo_core",
```

Comment:
I am still not sure if we need to add it (probably we need to)

Reply:
But it’s still shipped with puppet 6, and doesn’t this effectively mean you’re dropping support for anything other than >= 6 ?